source/cloud/azure/azure-ml-multi-node.md

```diff
@@ -0,0 +1,72 @@
+# Azure ML Compute cluster
+
+Launch Azure's [ML Compute cluster](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python) to distribute your RAPIDS training jobs across single or multi-GPU compute nodes. The Compute cluster scales up automatically when a job is submitted, and executes in a containerized environment, packaging your model dependencies in a Docker container.
+
+## Pre-requisites
+
+All you need to get started is an Azure machine learning workspace and development environment must be installed with [Python SDK v2](https://learn.microsoft.com/en-us/python/api/overview/azure/ai-ml-readme?view=azure-python)or [Azure CLI v2](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-cli?tabs=public). You could install Azure Machine Learning on your local computer but is recommended to use Azure's compute instance that is fully configured with all necessary packages and tools.
+
+### Instantiate workspace
+
+If using the Python SDK, connect to your workspace either by explicitly providing the workspace details or load from `config.json` file.
```

Comment:
Ah this is the point where we use the `config.json` file that is mentioned on the single node page.

Reply:
I don't actually see anywhere on the workspace page to download this file though (hope you don't mind me reusing your workspace). Am I missing it?

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/1610850/230330673-5b6b16c4-34ea-4231-b0cb-3a7cef27e685.png">

Reply:
the `config.json` file is downloaded from the workspace [homepage](ttps://portal.azure.com/?feature.msaljs=true#@NVIDIA.onmicrosoft.com/resource/subscriptions/fc4f4a6b-4041-4b1c-8249-854d68edcf62/resourceGroups/rapidsai-deployment/providers/Microsoft.MachineLearningServices/workspaces/rapids-aml-cluster/overview) in Azure ML portal (not the Studio)